pkg/tsdb/cloudwatch/models/settings.go

```diff
@@ -0,0 +1,37 @@
+package models
+
+import (
+	"encoding/json"
+	"fmt"
+
+	"github.com/grafana/grafana-aws-sdk/pkg/awsds"
+	"github.com/grafana/grafana-plugin-sdk-go/backend"
+)
+
+type CloudWatchSettings struct {
+	awsds.AWSDatasourceSettings
+	Namespace string `json:"customMetricsNamespaces"`
+}
+
+// Load is copied from grafana-aws-sdk -- json.Unmarshal was not loading the nested properties
```

Comment:
Huh, what's the issue with json.Unmarshal here...?

Could this bit use some tests? Could use it for the instance.Region / instance.Profile default values. And I am just thinking out loud, but a test could be a starting point to learn more about the json.Unmarshal issue mentioned in the comment above.

Reply:
Oh my bad - that comment is not valid anymore. Unmarshal is working just fine. Added a test.